form_utils/widgets.py

```diff
@@ -77,19 +129,33 @@ def render(self, name, value, attrs=None):
             self.value = value[0]
         else:
             self.value = value
-        return super(ClearableFileInput, self).render(name, value, attrs)
+
+        self.renderer.set_value(value)
+
+        render_kwargs = dict()
+        if django.VERSION > (1, 10, 0):
+            render_kwargs['renderer'] = self.renderer
```

Comment:
Alternatively, we could do this:

```python
render_kwargs['renderer'] = renderer
if django.VERSION < (1.11.0):
   del render_kwargs['renderer']
```

This way, the latest behavior is always the default, and when support is dropped for older versions, removing that code is easier.